operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -23,13 +23,26 @@ kiali_defaults:
 
   auth:
     strategy: ""
+    ldap:
+      ldap_host: ""
+      ldap_port: 0
+      ldap_use_ssl: false
+      ldap_insecure_skip_verify: false
+      ldap_user_filter: (cn=%s)
+      ldap_group_filter: (cn=%s)
+      ldap_base: ""
+      ldap_bind_dn: ""
+      ldap_search_filter: (&(name={USERID}))
+      ldap_role_filter: ""
+      ldap_mail_id_key: mail
+      ldap_user_id_key: cn
+      ldap_member_of_key: memberOf
 
   deployment:
     accessible_namespaces: ["^((?!(istio-operator|kube.*|openshift.*|ibm.*|kiali-operator)).)*$"]
     #additional_service_yaml:
     image_name: "kiali/kiali"
     image_pull_policy: "IfNotPresent"
-    image_pull_secrets: []
```

Comment:
I do not know why this was removed in this PR, it doesn't have anything to do with LDAP and this value must be specified explicitly as an empty list. Revert this change.

Reply:
This was unintended. Reverted this